src/common/RAJAPerfSuite.hpp

```diff
@@ -320,6 +332,95 @@ std::ostream* makeNullStream();
  */
 std::ostream& getNullStream();
 
+#if defined(RUN_KOKKOS)
```

Comment:
These things should be moved into a separate file. These things are specific to how kokkos variants manage kernel data. The RAJAPerfSuite.hpp file only should contain things needed for the define the suite contents (kernels, variants, etc.).